frameworks/kafka/src/main/dist/jmx-ssl-properties.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+JMX_SECURE_PROPERTIES_FILE=$MESOS_SANDBOX/.ssl/jmxremote.ssl.properties
+
+(umask u=rw,g=,o= && tee $JMX_SECURE_PROPERTIES_FILE <<EOF >/dev/null
```

Comment:
Why not just put the `umask` on a line of its own?
Also, I think you want `u=rwx,g=,o=`, otherwise we end up with a little unusual umask `0177`

Reply:
if you umask on a line of its own it will apply to everything which is coming later. 

the whole purpose of initial permission change was to don't have it execution permissions on it otherwise java will complain to set the correct permissions

Reply:
Right, this was kind of my point - there are just 2 commands in this file, and they both need the same umask set, no?

Reply:
Regading `rwx` - AFAIK files are never given execute permission by `O_CREAT` unless explicitly requested? It can stay as is, but I'd be somewhat confused to see an umask containing the octal digit `1` :-)